codecademy/functions.py

```diff
@@ -2,7 +2,7 @@
 def square(n):
     """This function returns the square of a number."""
     squared = n ** 2
-    print "%d squared is %d." % (n, squared)
+    print ("%d squared is %d." % (n, squared))
```

Comment:
good practice is not to have space between function name and the opening parenthesis
meaning
`print('hello')` is better then `print ('hello')`